app/src/main/res/values-zh-rCN/strings.xml

```diff
@@ -0,0 +1,380 @@
+    <string name="background_add_wrong_orientation">背景已添加,但它无法在所选方向中</string>
+    <string name="background_add_processing_failed">无法处理所选背景图片,请确定它未损坏.</string>
+    <string name="save_states_not_supported">运行固件时无法保存即时存档</string>
+    <string name="save_states_unavailable_ra_hardcore_enabled">硬核模式开启时无法保存即时存档</string>
+    <string name="rewind_time_seconds">%1$ss</string>
+    <string name="rewind_time_minutes_seconds">%1$dm%2$ss</string>
+    <string name="rewind_not_enabled">倒带未开启</string>
+    <string name="error_invalid_directory">无效目录</string>
+    <string name="error_invalid_directory_description">无法写入选择的目录,如果你正在选择一个压缩文件,请解压后再次尝试</string>
+    <string name="rewind">倒带</string>
+    <string name="quick_slot">快速保存</string>
+    <string name="saved">已保存</string>
+    <string name="loaded">已读取</string>
+    <string name="rewind_description">开启后会定时自动保存即时存档. 当一定时间内你需要回档,只需打开\"倒带\"界面选择要回档的即时存档即可,请注意,启用此功能会消耗大量存储空间, 而且频繁保存存档可能会造成软件卡顿,请酌情设置存档频率以及保存存档的时长.</string>
+    <string name="rewind_save_period">保存频率(每隔多久保存一次)</string>
+    <string name="rewind_length">倒带总长(会保留该时间内所有的存档)</string>
+    <string name="rewind_max_memory_usage">最大容量:%1$s</string>
+    <string name="rewind_memory_usage_above_recommended_limit">存储容量超过推荐的限制</string>
+    <string name="interpolation">插值</string>
+    <string name="bitrate">比特率</string>
+    <string name="input_quick_save">快速保存</string>
+    <string name="input_quick_load">快速读取</string>
+
+
+    <string-array name="fast_forward_speed_multiplier_options" tools:ignore="InconsistentArrays">
+        <item>无限制</item>
+        <item>1.5x</item>
+        <item>2x</item>
+        <item>3x</item>
+        <item>4x</item>
+        <item>8x</item>
+    </string-array>
+
+    <string-array name="rom_icon_filtering_options">
+        <item>无 (像素化)</item>
+        <item>Linear (线性流畅)</item>
+    </string-array>
+
+    <string-array name="game_runtime_console_type_options">
+        <item>默认</item>
+        <item>@string/console_ds</item>
+        <item>@string/console_dsi</item>
+    </string-array>
+
+    <string-array name="firmware_settings_language_options">
+        <item>英文</item>
+        <item>法语</item>
+        <item>德语</item>
+        <item>意大利语</item>
+        <item>西班牙语</item>
+        <item>日语</item>
+    </string-array>
+
+    <string-array name="game_runtime_mic_source_options">
+        <item>默认</item>
+        <item>无</item>
+        <item>吹气</item>
+        <item>系统麦克风</item>
+    </string-array>
+
+    <string-array name="console_type_options">
+        <item>@string/console_ds</item>
+        <item>@string/console_dsi</item>
+    </string-array>
+
+    <string-array name="fps_counter_position_options">
+        <item>隐藏</item>
+        <item>左上</item>
+        <item>顶部居中</item>
+        <item>右上</item>
+        <item>左下</item>
+        <item>底部居中</item>
+        <item>右下</item>
+    </string-array>
+
+    <string-array name="video_filtering_options">
+        <item>无</item>
+        <item>Linear</item>
```

Comment:
I believe this item, and the LCD one, can be translated. But unfortunately I don't know Chinese to know if that's that case or not, so I'd rather trust your judgement 😛